docs/source/api/interfaces.rst

```diff
@@ -13,8 +13,5 @@ External Interfaces
 .. automodule:: exatomic.interfaces.pdb
     :members:
 
-.. automodule:: exatomic.interfaces.tens
-    :members:
-
 .. automodule:: exatomic.interfaces.xyz
```

Comment:
Thanks for considering documentation! We should migrate to generating documentation using sphinx-apidoc (programmatic generation of rst files). Since that is outside the scope of this PR I've made an issue for this.